[2025-08-27 06:34:46] Starting Dual Adapter GRPO Training
Loading checkpoint shards: 100%|██████████| 4/4 [00:03<00:00,  1.17it/s]
[2025-08-27 06:34:51] Loading Data
[2025-08-27 06:34:51] Training samples: 1520, Validation samples: 190
[2025-08-27 06:34:51] === Data Samples ===
[2025-08-27 06:34:51] Sample Keys: ['id', 'input', 'output']
[2025-08-27 06:34:51] Sample Premise: 또한 ‘브람스를 좋아하세요?’는 SBS 단편드라마 ‘17세의 조건’으로 감각적 연출을 선보인 조영민 감독과 섬세한 필력을 자랑한 류보리 작가가 의기투합해 웰메이드 작품 탄생을 예고하고 있다. 박은빈과 김민재의 신선한 조합에, SBS 기대주 감독과 작가가 가세한 SBS 새 월화드라마 ‘브람스를 좋아하세요?’는 8월 31일에 첫 방송된다.
[2025-08-27 06:34:51] Sample Proposition: 박은빈이 출연한 SBS 월화 드라마의 제목에는 작곡가 이름이 들어간다.
[2025-08-27 06:34:51] Sample Label: entailment
[2025-08-27 06:34:51] Sample Reference Output: '브람스를 좋아하세요?'에 독일 작곡가인 브람스의 이름이 들어가 있다. 따라서 가설 문장은 함의에 해당한다.
[2025-08-27 06:34:51] ===================
[2025-08-27 06:34:51] Creating LoRA Adapters
/home/nlplab/anaconda3/envs/soeon/lib/python3.10/site-packages/peft/mapping_func.py:73: UserWarning: You are trying to modify a model with PEFT for a second time. If you want to reload the model with a different config, make sure to call `.unload()` before.
  warnings.warn(
/home/nlplab/anaconda3/envs/soeon/lib/python3.10/site-packages/peft/tuners/tuners_utils.py:190: UserWarning: Already found a `peft_config` attribute in the model. This will lead to having multiple adapters in the model. Make sure to know what you are doing!
  warnings.warn(
[2025-08-27 06:34:52] Training Adapter A(ROUGE Optimizer)
[2025-08-27 06:34:52] Reference Map Size: 1
Traceback (most recent call last):
  File "/media/nlplab/hdd1/hclt_ss/grpo/main.py", line 119, in <module>
    main()
  File "/media/nlplab/hdd1/hclt_ss/grpo/main.py", line 78, in main
    adapter_a = train_adapter_a(
  File "/media/nlplab/hdd1/hclt_ss/grpo/train.py", line 79, in train_adapter_a
    trainer = creat_grpo_trainer(
  File "/media/nlplab/hdd1/hclt_ss/grpo/train.py", line 36, in creat_grpo_trainer
    grpo_config = GRPOConfig(
  File "<string>", line 177, in __init__
  File "/home/nlplab/anaconda3/envs/soeon/lib/python3.10/site-packages/trl/trainer/grpo_config.py", line 604, in __post_init__
    raise ValueError(
ValueError: generation_batch_size (1) must be divisible by num_generations (8).
